pkg/connectors/policymanager/clients/policymanager_openapi.go

```diff
@@ -49,9 +50,22 @@ func NewOpenAPIPolicyManager(name, connectionURL string) (PolicyManager, error)
 
 func (m *openAPIPolicyManager) GetPoliciesDecisions(in *policymanager.GetPolicyDecisionsRequest,
 	creds string) (*policymanager.GetPolicyDecisionsResponse, error) {
-	resp, _, err := m.client.DefaultApi.GetPoliciesDecisions(context.Background()).XRequestCred(creds).GetPolicyDecisionsRequest(*in).Execute()
+	printErr := fmt.Sprintf("get policies decisions from %s failed", m.name)
```

Comment:
we use the message in the error cases, might be:

```golang
printErr := func() string { return fmt.Sprintf("get policies decisions from %s failed", m.name) }

if httpResponse == nil {
		if err != nil {
			return nil, errors.Wrap(err, printErr())
		}
		return nil, errors.New(printErr())
	}
```
Syntactic sugar :-)

Reply:
nice! changed to func